templates/trackers.html

```diff
@@ -1,7 +1,16 @@
 {% extends "base.html" %} 
-    {% block title %}
-		<title>Trackers</title>
-    {% endblock %}
+{% block title %}
+    <title>WhoTracks.me | Trackers</title>
+    <meta name="description" content="Top 500 most prevalent trackers on the web for a variety of categories such as advertising, analytics, customer interaction and others. More tracker profiles coming with each monthly release.">
```

Comment:
Can the 500 come from data (so it will auto-update if we add more data)?

I'd rewrite the text as `Top N most prevalent trackers on the web for advertising, analytics, customer interaction and others. Updated monthly.`

Reply:
Will change to: `Most prevalent trackers on the web for advertising, analytics, customer interaction and others. Updated monthly` - will not make a reference to the number.